awx/ui_next/src/screens/Template/JobTemplateEdit/JobTemplateEdit.jsx

```diff
@@ -21,14 +21,29 @@ class JobTemplateEdit extends Component {
     this.handleSubmit = this.handleSubmit.bind(this);
   }
 
-  async handleSubmit(values) {
+  async handleSubmit(values, newLabels, removedLabels) {
     const {
       template: { id, type },
       history,
     } = this.props;
 
+    const disassociatedLabels = removedLabels
```

Comment:
I don't love that we need to check these `*Labels` variables before using them / iterating over their contents.

These functions seem like a good candidate for default args, e.g: `handleSubmit(values, newLabels = [], removedLabels = [])`. By making their default values empty arrays, you don't need to check them before invoking `.forEach` - you can just do it.

Reply:
I didn't know that.  Thanks!